js/src/edit-block/components/header.js

```diff
@@ -29,7 +29,7 @@ const Header = () => {
 	return (
 		<div className="flex items-center h-16 border-b border-gray-300 w-full">
 			<a
-				className="flex items-center justify-center h-full w-16"
+				className="link-back-wp flex items-center justify-center h-full w-16"
```

Comment:
@kienstra I think you'll still need `bg-gray-900 text-white` here if you want white-on-black, unless you tweak the CSS separately. Up to you, though!

Reply:
Ah, great suggestion. Committed in https://github.com/studiopress/genesis-custom-blocks/pull/41/commits/ef0a4685245dc32e164ee58cf51cdf8d397bd729